.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 tison <wander4096@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- id: hawkeye-format
+  name: hawkeye-format
+  description: "Run `hawkeye format` for license header formatting"
+  entry: hawkeye format
+  language: python
+  args: []
+  pass_filenames: false
```

Comment:
I'd be nice if `hawkeye format` can take a list of filenames.

Reply:
We're currently using the `includes` option for passing files. It seems you mean an arbitrary position argument list that would be appended to the includes option?

Reply:
yeah, I think pre-commit usually pass changed filenames to the hook so it only needs to check these files which can be faster when doing `git commit`.

Reply:
IIRC `hawkeye format` is fast enough to handle more than 10,000 files in a few seconds. And if `excludes` configured properly, users should not feel different.

Reply:
yeah, it's not a blocker.